IFTPipeline.jl/src/cli.jl

```diff
@@ -51,6 +51,192 @@ function mkclipreprocess_single!(settings)
+        default = 200.0
+        arg_type = Float64
+        required = false
+
+        "--ice_labels_band_2_threshold"
+        default = 190.0
+        arg_type = Float64
+        required = false
+
+        "--ice_labels_ratio_lower"
+        default = 0.0
+        arg_type = Float64
+        required = false
+
+        "--ice_labels_ratio_upper"
+        default = 0.75
+        arg_type = Float64
+        required = false
+
+
+        # Adaptive histogram equalization parameters
+        
+        "--adapthisteq_white_threshold"
+        default = 25.5
+        arg_type = Float64
+        required = false
+
+        "--adapthisteq_entropy_threshold"
+        default = 4
+        arg_type = Float64
+        required = false
+
+        "--adapthisteq_white_fraction_threshold"
+        default = 0.4
+        arg_type = Float64
+        required = false
+
+
+        # Gamma parameters
+        
+        "--gamma"
+        default = 1
+        arg_type = Float64
+        required = false
+
+        "--gamma_factor"
+        default = 1
+        arg_type = Float64
+        required = false
+
+        "--gamma_threshold"
+        default = 220
+        arg_type = Float64
+        required = false
+
+
+        # Unsharp mask parameters
+        
+        "--unsharp_mask_radius"
+        default = 10
+        arg_type = Int
+        required = false
+
+        "--unsharp_mask_amount"
+        default = 2.0
+        arg_type = Float64
+        required = false
+
+        "--unsharp_mask_factor"
+        default = 255.0
+        arg_type = Float64
+        required = false
+
+
+        # Brighten parameters
+        
+        "--brighten_factor"
+        default  = 0.1
+        arg_type = Float64
+        required = false
+
+
+        # Preliminary ice mask parameters
+        
+        "--prelim_icemask_radius"
+        default = 10
+        arg_type = Int
+        required = false
+
+        "--prelim_icemask_amount"
+        default = 2
+        arg_type = Int
+        required = false
+
+        "--prelim_icemask_factor"
+        default = 0.5
+        arg_type = Float64
+        required = false
+
+
+        # Main ice mask parameters
+        
+        "--icemask_band_7_threshold"
+        default = 5
+        arg_type = Int
+        required = false
+
+        "--icemask_band_2_threshold"
+        default = 230
+        arg_type = Int
+        required = false
+
+        "--icemask_band_1_threshold"
+        default = 240
+        arg_type = Int
+        required = false
+
+        "--icemask_band_7_threshold_relaxed"
+        default = 10
+        arg_type = Int
+        required = false
+
+        "--icemask_band_1_threshold_relaxed"
+        default = 190
+        arg_type = Int
+        required = false
+
+        "--icemask_possible_ice_threshold"
+        default = 75
+        arg_type = Int
+        required = false
+
+        "--icemask_n_clusters"
+        default = 3
+        arg_type = Int
+        required = false
```

Comment:
Have you considered using a configuration file? The CLI is getting quite complex with many options.